images/php/configs/fpm.apko.yaml

```diff
@@ -0,0 +1,46 @@
+contents:
```

Comment:
Do we want to name this file `latest-fpm.apko.yaml` so it produces the tag `:latest-fpm` (and `-dev`) to match the previous tag?